background/main.ts

```diff
@@ -1845,6 +1849,7 @@ export default class Main extends BaseService<never> {
   ): Promise<{
     asset: SmartContractFungibleAsset
     amount: bigint
+    mainCurrencyAmount: number | undefined
```

Comment:
I remember that we talked about the convention that we should use the following version.
```suggestion
    mainCurrencyAmount?: number
```